src/tvdb/client.py

```diff
@@ -178,12 +188,18 @@ async def fetch(
             raise BadCallError("Meta can only be used with extended=True.")
         elif short:
             raise BadCallError("Short can only be enabled with extended=True.")
-        response = await client.request(
-            "GET",
-            f"{cls.ENDPOINT}/{media_id}" + ("/extended" if extended else ""),
-            query=query if query else None,
-        )
+        if not response:
+            response = await client.request(
+                "GET",
+                f"{cls.ENDPOINT}/{media_id}" + ("/extended" if extended else ""),
+                query=query if query else None,
+            )
+            await cache.set(key=cache_key, value=response)
```

Comment:
No time-to-live? Can we afford to potentially having to cache tens of thousands of these?

(Not a direct request to add ttl here, more of a discussion prompt, since I'm not sure)

Reply:
I don't think we need to care about that. In my head, i was going to implement redis in that, and there's no need for ttl in memory since it's supposed to only be used while doing developement. But maybe it's till a good idea.

Reply:
ttl does work with the memory backend for aiocache out of the box, I checked, it's just scheduling asyncio tasks into the event loop that then run the delete func. It's not like this would require any special implementations, and the call would be exactly the same once/if we switch to redis. Since `BaseCache.get` has the `ttl` arg defined, subclasses for individual backends just implement the abstract `_get` which gets called here.